### PR TITLE
Fix/215 toleration on hook

### PR DESCRIPTION
**Pull request by Jufik**
Please ensure your pull request adheres to the following guidelines:
- [x] explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [x] Document your Changes in the `CHANGELOG.md` file of the respected chart as well as the `Chart.yaml`
- [x] Bump the Version number of the respected chart

Fix for [issue 215](https://github.com/SonarSource/helm-chart-sonarqube/issues/215), 
adding tolerations and affinity to Hooks

### CODE REVIEW

**Comment by lukasz-jarocki-sonarsource** [comment]
Hi @leo-geoffroy-sonarsource  can you take a look at this one?

**Comment by leo-geoffroy-sonarsource** [comment]
@Jufik 
Thank you for your contribution.
This change does not qualify for the LTS, could you please remove the changes from the sonarqube-lts folder?

File: charts/sonarqube-dce/templates/change-admin-password-hook.yml [hunk 1]
```diff
@@ -41,6 +41,14 @@ spec:
         {{- end }}
       {{- end }}
       serviceAccountName: {{ template "sonarqube.serviceAccountName" . }}
+    {{- if .Values.tolerations }}
+      tolerations:
+{{ toYaml .Values.tolerations | indent 8 }}
+    {{- end }}
+    {{- if .Values.affinity }}
+      affinity:
+{{ toYaml .Values.affinity | indent 8 }}^
```
Comment (leo-geoffroy-sonarsource) [inline charts/sonarqube-dce/templates/change-admin-password-hook.yml:50]: I think there is a typo at the end ```}}^```

Comment (Jufik) [reply charts/sonarqube-dce/templates/change-admin-password-hook.yml]: OK Fixed!

**Comment by Jufik** [comment]
@leo-geoffroy-sonarsource : removed LTS modifications

**Comment by leo-geoffroy-sonarsource** [comment]
LGTM, I just fixed a typo and squashed the commit 
Thanks again for the contribution

**Pull request merged**